pages/app/controllers/refinery/pages_controller.rb

```diff
@@ -101,7 +104,7 @@ def action_page_finder
     module Finders
       class Home
         def self.call(_params)
-          Refinery::Page.find_by link_url: "/"
+          Refinery::Page.find_by link_url: Refinery::Pages.home_page_path || "/"
```

Comment:
Do we need a fallback to `"/"` if the `Refinery::Pages.home_page_path` config has `"/"` as default ?

Reply:
Nope. Just making sure it happens somewhere.